.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -33,11 +33,11 @@ assignees: ''
 
 **After release**:
 - [ ] Update conda-forge [pygmt-feedstock](https://github.com/conda-forge/pygmt-feedstock) [Usually done automatically by conda-forge's bot]
-- [ ] Bump PyGMT version on https://github.com/GenericMappingTools/try-gmt
+- [ ] Bump PyGMT version on https://github.com/GenericMappingTools/try-gmt (after conda-forge update)
 - [ ] Announce the release on:
-  - [ ] GMT [forum](https://forum.generic-mapping-tools.org/c/news/)
+  - [ ] GMT [forum](https://forum.generic-mapping-tools.org/c/news/) (requires moderator status)
```

Comment:
Should we also mention that we draft the forum post on https://hackmd.io/@pygmt?

```suggestion
  - [ ] GMT [forum](https://forum.generic-mapping-tools.org/c/news/) (do this first! requires moderator status)
```